### ISSUE TITLE
igx-grid cells aria-describedby is pointing to an id that doesn't exist

### ISSUE THREAD

**Issue by kdinev** [IgniteUI/igniteui-angular#497]
### Description  
The `igx-grid` `th` elements don't have ids.

### Steps to reproduce  

1. Open https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid.html
2. Inspect any grid cell and its `aria-describedby` attribute
3. Inspect the `th` the grid cell belongs to

### Result  
There is no id on the `th` element that corresponds to the `aria-describedby` attribute value.

### Expected result  
The `th` element id should be the same as the `aria-describedby` value.

### Attachments  
![image](https://user-images.githubusercontent.com/1472513/34603009-c936b06e-f20a-11e7-92ca-8c62e6f89c8e.png)
